FinSearchUnified/Tests/Helper/UrlBuilderTest.php

```diff
@@ -21,20 +23,6 @@ class UrlBuilderTest extends TestCase
      */
     private $httpResponse;
 
-    public function setUp()
```

Comment:
The actual idea was to keep this set up method and just remove the `...$httpClientMock->expects($this->atLeastOnce())...` part, e.g.
```
    public function setUp()
    {
         $this->httpClient = $this->getMockBuilder(Zend_Http_Client::class)
            ->setMethods(['request'])
            ->getMock();
    }
```

And in the tests you will have to create a response object and write your expectations, e.g. `$this->httpClient->expects....`